sites/partners/package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "NODE_OPTIONS='--inspect=9231' next -p ${NEXTJS_PORT:-3001}",
     "build": "next build",
     "test": "concurrently \"yarn dev\" \"cypress open\"",
-    "test:unit": "jest -w 1",
+    "test:unit": "jest -w 1 --detectOpenHandles --forceExit",
```

Comment:
Do you mind explaining why this addition was needed?

Reply:
My b these are flags I added to figure out why the performance running locally was so bad, I'm not 100% sure what it'll do to the performance in CI but I removed em and will take a look 👁️